config/initializers/100-sidekiq.rb

```diff
@@ -124,3 +93,32 @@ def call(ex, context = {})
 Sidekiq.error_handlers << SidekiqLogsterReporter.new
 
 Sidekiq.strict_args!
+
+Rails.application.config.to_prepare do
+  # Ensure that scheduled jobs are loaded before mini_scheduler is configured.
+  if Rails.env.development?
+    Dir.glob("#{Rails.root}/app/jobs/scheduled/*.rb") do |f|
+      require_dependency(f)
```

Comment:
Should we be using `require_dependency` here when the method is obsolete? 

> Warning: This method is obsolete. The semantics of the autoloader match Ruby's and you do not need to be defensive with load order anymore. Just refer to classes and modules normally.

Reply:
Since this is related to jobs, I’ll have to check.